github/event_types.go

```diff
@@ -228,6 +228,16 @@ type Discussion struct {
 	Body               *string             `json:"body,omitempty"`
 }
 
+// DiscussionCommentEvent represents a discussion comment event.
+type DiscussionCommentEvent struct {
+	Action     *string            `json:"action,omitempty"`
+	Comment    *DiscussionComment `json:"comment,omitempty"`
+	Discussion *Discussion        `json:"discussion,omitempty"`
+	Repo       *Repository        `json:"repository,omitempty"`
+	Org        *Organization      `json:"organization,omitempty"`
+	User       *User              `json:"user,omitempty"`
```

Comment:
```suggestion
	Sender       *User              `json:"sender,omitempty"`
```

The field names must match the official documentation, obviously. I do not have time for a full review right now. Please verify for yourself that your solution is correct before I perform a full review.